crates/forge/tests/integration/elect.rs

```diff
@@ -154,7 +154,6 @@ fn elect_complex() {
                 let elect_checker1 = IElectCheckerDispatcher { contract_address: contract.deploy(@ArrayTrait::new()).unwrap() };
                 let elect_checker2 = IElectCheckerDispatcher { contract_address: contract.deploy(@ArrayTrait::new()).unwrap() };
 
-                let old_seq_addr1 = elect_checker1.get_sequencer_address();
```

Comment:
@drknzz This variable was never used. Should it be removed or maybe the test should be updated?

Reply:
The test is okay, it should be removed 👍